Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -713,13 +713,15 @@ void WebChromeClient::unavailablePluginButtonClicked(Element& element, RenderEmb
 void WebChromeClient::mouseDidMoveOverElement(const HitTestResult& hitTestResult, unsigned modifierFlags, const String& toolTip, TextDirection)
 {
     RefPtr<API::Object> userData;
+    
+    auto modifiers = OptionSet<WebEvent::Modifier>::fromRaw(modifierFlags);
```

Comment:
Just to follow-up on this patch—everything was correct except for this one change.

The `modifierFlags` parameter contains an `NSEventModifierFlags` value on macOS, and  `WebEventFlags` value on iOS (and any platform with UIKit mouse support), so this requires a separate conversion method instead of using `OptionSet<WebEvent::Modifier>::fromRaw()`.